src/main/java/ro/msg/learning/shop/controller/ProductController.java

```diff
@@ -0,0 +1,52 @@
+package ro.msg.learning.shop.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Again the same remark regarding the imports: do not use *, import the resources one by one